### ISSUE TITLE
Does not work in shadow DOM

### ISSUE THREAD

**Issue by configurator** [rakannimer/react-google-charts#322]
When used inside any element's shadow DOM, the charts don't work.
This is because they rely on `document.getElementById()`, which does not penetrate shadow DOMs.

Simple demo:
```jsx
import React from 'react';
import Shadow from 'react-shadow';
import Chart from 'react-google-charts';

export default () => (
  <Shadow.div>
    <Chart width="100px" height="150px" chartType="Bar" data={['Title', 'Value'], ['A', 100], ['B', 200]]} />
  </Shadow.div>
);
```

This component will fail to display a chart, with the error `Uncaught Error: The container #reactgooglegraph-XXX is not defined.`

**Comment by mbrevda** [comment]
Noticed the same with [Portals](https://codesandbox.io/s/intelligent-mccarthy-5e5m7?file=/index.js). Is this a bug in the lib @rakannimer?

**Comment by mbrevda** [comment]
To clarify, my issue was specifically when using an iframe in a portal.

Superstart @tikotzky 🚀  pointed me in the right direction: the charting lib takes an id, and then searches for the dom element with that id in the current `document`. The issue was that code is being executed on the host document while the dom nodes are being drawn in the iframe. So when Google Charts does `document.getElementById`, it cannot find the dom nodes as it looking in the wrong `document`. Unfortionalty, it doesn't look like you can pass a dom node or a browser context to Google Charts.

**tl;dr:** Google Charts is showing it's age and doesn't seem to be well maintained anymore. This is not an issue with react-google-charts (albeit the latter might be suffering from a similar ailment...)
